stats/stats/src/charts/mod.rs

```diff
@@ -70,3 +74,72 @@ pub async fn create_chart(
     .await?;
     Ok(())
 }
+
+#[async_trait]
+pub trait ChartFullUpdater: Chart {
+    async fn get_values(
+        &self,
+        blockscout: &DatabaseConnection,
+    ) -> Result<Vec<DateValue>, UpdateError>;
+
+    async fn update_with_values(
+        &self,
+        db: &DatabaseConnection,
+        blockscout: &DatabaseConnection,
+        _full: bool,
```

Comment:
why does ChartFullUpdater need _full: bool? just to match signature of ChartUpdater?

Reply:
yes, it will be easier to replace one to another, just change the trait, don't change the update function
both is fine imo, but here I kinda prioritized easier to refactor/change